src/open_inwoner/accounts/tests/test_profile_views.py

```diff
@@ -703,16 +730,12 @@ def test_wrong_date_format_shows_birthday_none_brp_v_1_3(self, m):
             },
         )
         response = self.app.get(self.url, user=self.user)
-        log_entry = TimelineLog.objects.last()
 
         self.assertIsNone(response.context["my_data"].birthday)
-        self.assertEqual(
-            log_entry.extra_data,
-            {
-                "message": _("user requests for brp data"),
-                "action_flag": list(LOG_ACTIONS[4]),
-                "content_object_repr": self.user.email,
-            },
+        self.assertTimelineLog(
+            _("user requests for brp data"),
+            content_object_repr=self.user.email,
+            action_flag=list(LOG_ACTIONS[4]),
```

Comment:
This exists because we want less improvised add-hoc code in the tests, and log tests that don't just take `.last()`. 

We can rename it to assertTimelineLogExists if that is any better.